pkg/minikube/registry/registry_test.go

```diff
@@ -39,41 +53,62 @@ func TestRegistry(t *testing.T) {
 	}
 
 	registry := createRegistry()
+	t.Run("registry.Register", func(t *testing.T) {
+		t.Run("foo", func(t *testing.T) {
+			if err := registry.Register(foo); err != nil {
+				t.Fatal("expect nil")
```

Comment:
`Error not expected but got %v` -> we want to know what was the error

Reply:
fixed.